data/json/items/ammo.json

```diff
@@ -1178,6 +1178,86 @@
     "ammo_type": "components",
     "count": 150
   },
+  {
+    "type": "AMMO",
+    "id": "lc_chain_link",
```

Comment:
These links can probably all use copy-from.

Reply:
Oh yep, I tried toc clean up with copy from but missed this one.